app/src/main/res/values-pcm/strings.xml

```diff
@@ -0,0 +1,511 @@
+  <string name="view_all">View All</string>
+  <string name="ongoing_story_last_week">You play am last week</string>
+  <string name="ongoing_story_last_month">You play am last month</string>
+  <string name="story_activity_title">Chapter List</string>
+  <string name="lesson_thumbnail_content_description">Picture for %s</string>
+  <string name="stories_you_can_play">Stories Wey You Fit Play</string>
+  <string name="navigate_up">Go up</string>
+  <string name="recently_played_activity">Stories wey you just play</string>
+  <string name="topic_downloaded">Topic Wey You Don Download</string>
+  <string name="downloaded">You don download am</string>
+  <string name="question_player_title">Practice Mode</string>
+  <string name="question_training_session_progress">Question %s of %s</string>
+  <string name="question_training_session_progress_finished">Complete</string>
+  <string name="question_training_session_finished_header">Finished</string>
+  <string name="question_training_session_finished_message">You don finish all of di questions! You fit choose to play anoda set of questions, or go back to di topic.</string>
+  <string name="status_in_progress">In Progress</string>
+  <string name="status_completed">Completed</string>
+  <string name="show_chapter_list">Show di chapter list</string>
+  <string name="hide_chapter_list">Hide di chapter list</string>
+  <string name="play_pause_audio">Play/Pause Audio</string>
+  <string name="preferences">How you like am</string>
+  <string name="profile_progress_activity_title">Profile Progress Page</string>
+  <string name="search">Find-am</string>
+  <string name="fraction_error_invalid_chars">Abeg only use numerical digits, spaces or forward slashes (/)</string>
+  <string name="fraction_error_invalid_format">Abeg enter a valid fraction (e.g., 5/3 or 1 2/3)</string>
+  <string name="fraction_error_divide_by_zero">Abeg no put 0 for di denominator</string>
+  <string name="fraction_error_larger_than_seven_digits">None of di nombas for di fraction go get more dan 7 digits.</string>
+  <string name="number_error_starting_with_floating_point">Abeg start your ansa with nomba (e.g.,”0” for 0.5).</string>
+  <string name="number_error_invalid_format">Abeg put correct nomba.</string>
+  <string name="number_error_larger_than_fifteen_characters">Di ansa fit get at most 15 digits (0–9) or sign (. or -).</string>
+  <string name="ratio_error_invalid_chars">\n    Abeg write a ratio wey get nomba separated by colons (e.g. 1:2 or 1:2:3).</string>
+  <string name="ratio_error_invalid_format">\n    Abge enter a valid ratio (e.g. 1:2 or 1:2:3).</string>
+  <string name="ratio_error_invalid_colons">\n    Your ansa get two colons (:) next to each other.</string>
+  <string name="ratio_error_invalid_size">\n    Nomba of terms no dey equal to di required terms.</string>
+  <string name="ratio_error_includes_zero">\n    Ratios no suppose get 0 for inside.</string>
```

Comment:
In the main strings file, these strings span more than one line, hence the [incorrect] insertion of the newline.